src/content/reference/kubectl-gs/get-nodepools.md

```diff
@@ -0,0 +1,94 @@
+---
+title: "'kubectl gs get nodepools' command reference"
+description: Reference documentation on how to list node pools and get details for a single node pools using 'kubectl gs'.
+type: page
+weight: 10
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-ux
+---
+
+# `kubectl gs get nodepools`
+
+{{% kgs_alias_assumption %}}
+
+Like with all `get` commands in `kubectl`, this command can be used to get details on one item, a node pool in this case, or list several of them.
+
+## Usage
+
+### Get a list of node pools {#list}
+
+Simply execute
+
+```nohighlight
+kgs get nodepools
+```
+
+to list some information on all node pools available to you in the current installation.
+
+Here is some example output:
+
+```nohighlight
+ID      CLUSTER ID   CREATED                         CONDITION   NODES MIN/MAX   NODES DESIRED   NODES READY   DESCRIPTION
+ab12c   s921a        2021-01-02 15:04:32 +0000 UTC   READY       3/10            5               3             Production node pool
+```
+
+### Get specific node pool
+
+When used with a node pool ID as additional argument, the command will show details for a single node pool. Example:
+
+```nohighlight
+kgs get nodepools ab12c
+```
+
+Note: As an alternative to `get nodepools`, `get nodepool` will also work.
+
+## Output {#columns}
+
+The standard tabular output format features these columns:
+
+- `ID`: Unique identifier of the node pool.
+- `CLUSTER ID`: Unique identifier of the cluster that the node pool belongs to.
+- `CREATED`: Date and time of the node pool CR creation.
+- `CONDITION`: Latest condition reported for the node pool.
```

Comment:
As explained in https://github.com/giantswarm/kubectl-gs/pull/254#issuecomment-760955591, this is not available on AWS. That could be mentioned here. Also we could explain what `n/a` could mean on Azure.